servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -164,6 +169,12 @@ def _auth_token():
 
 def _generate_dstat_jobs(provider, query, create_time_max=None,
                          offset_id=None):
+    # If create_time_max is not set, but we have to client-side filter by
+    # end-time, set create_time_max = query.end because all the jobs with
+    # create_time >= query.end cannot possibly match the query.
+    if not create_time_max and query.start:
```

Comment:
hmmm.. I thought this was supposed to be query.end?

Do the tests fail?

Reply:
Whoops you're definitely should be `query.end`, should've run the tests again to make sure they still pass.